src/main/resources/templates/delete.ftl

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<body>
+<h3>Blog deletion</h3>
+<a href="http://localhost:8080/">Home</a>
```

Comment:
navigation bar items currently are hardcoded instead you need to place them in one separated template with a name like "navbar.tfl" and then you can import/include it on every page
see for details:
https://freemarker.apache.org/docs/ref_directive_include.html
https://freemarker.apache.org/docs/ref_directive_import.html